packages/testlab/test/integration/test-sandbox.integration.ts

```diff
@@ -39,24 +47,59 @@ describe('TestSandbox integration tests', () => {
   it('copies a file to the sandbox', async () => {
     await sandbox.copyFile(COPY_FILE_PATH);
     expect(await pathExists(resolve(path, COPY_FILE))).to.be.True();
-    await compareFiles(resolve(path, COPY_FILE));
+    await compareFiles(COPY_FILE_PATH, resolve(path, COPY_FILE));
   });
 
   it('copies and renames the file to the sandbox', async () => {
     const rename = 'copy.me.js';
     await sandbox.copyFile(COPY_FILE_PATH, rename);
     expect(await pathExists(resolve(path, COPY_FILE))).to.be.False();
     expect(await pathExists(resolve(path, rename))).to.be.True();
-    await compareFiles(resolve(path, rename));
+    await compareFiles(COPY_FILE_PATH, resolve(path, rename));
   });
 
   it('copies file to a directory', async () => {
     const dir = 'test';
-    await sandbox.mkdir(dir);
     const rename = `${dir}/${COPY_FILE}`;
     await sandbox.copyFile(COPY_FILE_PATH, rename);
     expect(await pathExists(resolve(path, rename))).to.be.True();
-    await compareFiles(resolve(path, rename));
+    await compareFiles(COPY_FILE_PATH, resolve(path, rename));
+  });
+
+  it('copies source file and map to directory', async () => {
+    await sandbox.copySourceCodeFile(TEST_FILE_RESOLVED);
+    expect(await pathExists(resolve(path, TEST_FILE))).to.be.True();
+    expect(await pathExists(resolve(path, TEST_FILE_MAP))).to.be.True();
```

Comment:
These assertions will produce error messages that are rather unhelpful ("expected false to be true"). Please create a helper for verifying that a path exists, one that will include the file path in the error message.

For example:

```ts
async expectPathExists(filePath: string) {
  expect.ok(
    await pathExists(filePath),
   `Expected "${filePath}" to exist.`);
}
```